core/domain/acl_decorators.py

```diff
@@ -1615,7 +1615,7 @@ def can_delete_question(handler):
     """Decorator to check whether the user can delete a question.
 
     Args:
-        handler. The function to be decorated.
+        handler: The function to be decorated.
```

Comment:
Technically, should have typeinfo, i.e.:

    handler: function. The function to be decorated.

Ditto below.